pkg/skuba/actions/node/remove/remove.go

```diff
@@ -62,6 +48,22 @@ func Remove(client clientset.Interface, target string, drainTimeout time.Duratio
 
 	var isControlPlane bool
 	if isControlPlane = kubernetes.IsControlPlane(node); isControlPlane {
+		cp, err := kubernetes.GetControlPlaneNodes(client)
```

Comment:
The same as above. I would suggest to update node replica and affinity when node change no matter what master/worker node is.

Reply:
Thanks